src/components/PersonLink.tsx

```diff
@@ -0,0 +1,13 @@
+import { Person } from '../types';
+
+interface PersonLinkProps {
+  person: Person
+}
+
+export const PersonLink: React.FC<PersonLinkProps> = ({ person }) => {
+  return (
+    <div>
+      {person.name}
+    </div>
+  );
+};
```

Comment:
lets create some structure

create `pages ` folder and there will be `HomePage.tsx` and `PeoplePage.tsx`, `PageNotFound.tsx`

folder `components ` use for components which we use for our pages